projects/ccf-rui/src/app/shared/components/tag-search/tag-search.component.scss

```diff
@@ -1,97 +1,136 @@
 :host {
   display: block;
+  position: relative;
 
-  mat-form-field {
-    width: 100%;
-
-    mat-select {
-      cursor: text;
-
-      mat-select-trigger {
-        .icon.add {
-          position: absolute;
-          // Position determined by experimentation
-          top: -1.1875rem;
-          right: -1.1875rem;
-          width: 3.625rem;
-          height: 3.625rem;
-          font-size: 3.625rem;
-          cursor: pointer;
-        }
+  .spacer {
+    // Calculated by adding up all padding/margin/height of material form fields
+    height: 3.25rem;
+  }
+
+  .overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    // box-shadow: 0rem 0rem 0.3rem 0.1rem #9e9e9e;
+
+    .search-box {
+      display: flex;
+      width: 100%;
+
+      .add-button {
+        width: 1.5rem;
+        height: 1.5rem;
+        line-height: 1.5rem;
       }
     }
-  }
 
-  ::ng-deep .mat-form-field-outline-thick {
-    opacity: 1 !important;
-  }
+    ::ng-deep {
+      .mat-form-field-wrapper {
+        padding-bottom: 0;
+      }
 
-  ::ng-deep .mat-form-field-infix {
-    height: 3rem;
-    border-top: 0;
-    cursor: text;
+      .mat-form-field-infix {
+        padding: 0 0 0.5rem 0;
+      }
+    }
   }
 
-  ::ng-deep .mat-select-trigger {
-    width: 100%;
-    cursor: text;
-  }
+  // mat-form-field {
```

Comment:
Yipes! That's a lot of commented out code. Do we need it? What's the story here?

Reply:
Forgot to remove it